PermanentOrg/web-app
# PER-8903 pass group instead of family

Change all the instances of family to group. Now the frontend handles group archives, creating dispalying editing. This will work once the back-end is done.

Steps to test: 
1. Open devtools network tab.
2. Create or change the type of an archive to be a group.
3. The request should be successful .
